src/components/common/CardView/CardView.jsx

```diff
@@ -22,7 +23,8 @@ const CardView = (props) => {
     onClickProfileImage,
   } = props;
   return (
-    <div style={cardDetailsContainerStyle} onClick={() => { onCardClick(cardDetails); }}>
+    <div onClick={() => { onCardClick(cardDetails) }} className="cardDetailsContainerStyle">
+      <img className="card-img-absolute" src={LIST_CARD_BG_WATERMARK} alt="circle-image"/>
```

Comment:
Please create a separate React component for this with name **CardBackgroundImage**
```suggestion
      <img className="card-img-absolute" src={LIST_CARD_BG_WATERMARK} alt="circle-image"/>
```

Reply:
done